app/controllers/o_data_controller.rb

```diff
@@ -13,6 +13,11 @@ class ODataController < ApplicationController
 
   skip_before_action :verify_authenticity_token, only: :options
 
+  # This action needs to register first.
+  # Clients may override the method if they want to do something.
+  before_action :before_action
+  def before_action; end
```

Comment:
🧐 
I don't get this one... Quite confusing. Could we call it something other than `before_action`?

Reply:
To use this in NEMO, we open-class this controller and re-define the method called `before_action` to actually do something (instead of a noop like it is here).

The "normal" way to do it with `before_action :foo` doesn't work for us because the open-class's `before_action`s is executed AFTER all of the original class's `before_action`s have run, which causes errors because several actions depend on the schema existing FIRST.

Any naming advice? It's a strange situation that I'm not sure if Rails has a solution for... I wanted to maintain clarity that it runs the same way a `before_action :foo` would.

Ideas:
- `before_all_actions`
- `before_action_hook`